Value jump indicator

friedmud opened the pull request:
Just adding a `ValueJumpIndicator` for completeness

closes #10462

friedmud committed:
Add ValueJumpIndicator refs #10462

friedmud committed:
More work on VJI #10462

friedmud committed:
Finish test for VJI closes #10462

moosebuild commented:
Job [Documentation](https://www.moosebuild.org/job/143988/) on 505f146 wanted to post the following:

View the site [here](http://mooseframework.org/docs/PRs/10474/site)

This comment will be updated on new commits.

lindsayad commented:
Also should close duplicate #6013

permcody commented:
@jwpeterson - assigning to you since you had several comments on #6013 and you found issues with the last indicator changes that were proposed.

jwpeterson committed:
Add documentation stub.

Refs #10462.

jwpeterson commented:
I added a stub markdown file for the new class.
```
moose/docs/content/documentation/systems/Adaptivity/Indicators/framework/ValueJumpIndicator.md
```

jwpeterson committed:
Add some actual documentation.

Forgot that CIVET does not allow new classes to be added without any documentation.

Refs #10462.
